### DISCUSSION TITLE
pb_public using  Systemd service file.

### DISCUSSION THREAD

**Discussion by qdubois** [pocketbase/pocketbase#470] [Q&A] [answered]
Hello,

First of all thanks for the great work :)

when I want to use the pb_public using when setting up a Systemd service file.
I can access to the static files when launching pocketbase maunually, but it does not when launching it via a service.
I can access to pb_data without any issue. But when launched via a service it does not work.

I tried to add WorkingDirectory but it did not change anything.

Thanks

**Comment by ganigeorgiev** [comment, marked as answer]
It should work if `WorkingDirectory` is pointing to the directory where the executable is located.

I've just tested it with a new Ubuntu vm with the following dir structure:
```
/root/pb/
  - pocketbase
  - pb_data/
  - pb_public/
    - index.html
```

I'm using the following service file:

```nginx
[Unit]
Description = pocketbase

[Service]
Type             = simple
User             = root
Group            = root
LimitNOFILE      = 4096
Restart          = always
RestartSec       = 5s
StandardOutput   = append:/root/pb/errors.log
StandardError    = append:/root/pb/errors.log
WorkingDirectory = /root/pb/
ExecStart        = /root/pb/pocketbase serve --http="YOUR_IP_OR_DOMAIN:80"

[Install]
WantedBy = multi-user.target
```

And I've started the service by just running `systemctl start pocketbase`.

_In the next bugfix release v0.7.3 (sometime tomorrow) I'll try to change the prebuilt binary to load the `pb_public` directory relatively to the binary file, so that we don't have to rely on the `WorkingDirectory` setting._

**Reply by qdubois** [reply]
it worked !!
thanks a lot
